evolver/settings.py

```diff
@@ -34,11 +34,27 @@ class Settings(BaseSettings):
 
 
 class AppSettings(BaseSettings):
-    CONFIG_FILE: Path = Path("evolver.yml")  # in current directory
+    def find_project_root(self, marker: str = "pyproject.toml") -> Path:
+        """
+        Find the project root by looking for a marker file in parent directories.
+        Defaults to "pyproject.toml".
+        """
+        path = Path(__file__).resolve()
+        for parent in path.parents:
+            if (parent / marker).exists():
+                return parent
+        raise FileNotFoundError(f"Project root marker '{marker}' not found in any parent directories.")
+
+    CONFIG_FILE: Path | None = None
     LOAD_FROM_CONFIG_ON_STARTUP: bool = True
     HOST: str = "127.0.0.1"
     PORT: int = 8080
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.CONFIG_FILE is None:
+            self.CONFIG_FILE = self.find_project_root() / "evolver.yml"
```

Comment:
This is already what the app will do, kinda, assuming that it's started from the repo dir.

Reply:
yeah but not if I am using a debugger and start from a module, e.g. evolver.app.main, in that case the working dir for the process is 2 dirs deep relative to the project root, so it can't find the evolver.yml.

Reply:
You should change the working dir in your debugger